src/app/blog/iroh-content-discovery/page.mdx

```diff
@@ -0,0 +1,255 @@
+import { BlogPostLayout } from '@/components/BlogPostLayout'
+import Image from 'next/image'
+
+export const post = {
+  draft: false,
+  author: 'Rüdiger Klaehn',
+  date: '2024-06-27',
+  title: 'Iroh content discovery',
```

Comment:
I think the title is a bit vague, maybe sth like "content discovery explorations" or in that direction